save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/ProjectFilters.kt

```diff
@@ -1,11 +1,19 @@
 package com.saveourtool.save.filters
 
+import com.saveourtool.save.entities.ProjectStatus
 import kotlinx.serialization.Serializable
 
 /**
  * @property name
+ * @property status
  */
 @Serializable
-class ProjectFilters(
-    val name: String?,
-)
+data class ProjectFilters(
+    val name: String,
+    val status: List<ProjectStatus> = listOf(ProjectStatus.CREATED),
```

Comment:
```suggestion
    val statuses: Set<ProjectStatus> = EnumSet.of(ProjectStatus.CREATED),
```

Reply:
I can't resolve ```EnumSet```

Reply:
`java.util.EnumSet`

Reply:
Thanks.
Returning to the previous problem - java is not available on the frontend.